.rr.yml

```diff
@@ -0,0 +1,9 @@
+http:
+  address: ":8080"
+  workers:
+    command: "php psr-worker.php"
+    pool:
+      numWorkers: 1
```

Comment:
Session currently has issues with more workers. Seems it isn't shared by them.

Reply:
Fixed with https://github.com/yiisoft/yii-web/commit/4876355e6a3cf220eb281d49768aa7ac1921f792